### PR TITLE
common: non-gps navigation with Luxonis OAK-D

### PR DESCRIPTION
**Pull request by chobitsfan**
It explains how to setup non-gps navigation with Luxonis OAK-D. 
https://discuss.ardupilot.org/t/a-vio-system-using-oak-d-raspberry-pi/112761/1

### CODE REVIEW

**Comment by Hwurzburg** [comment]
Once @rmackay9 approves, II will fix the build errors and tweak the formatting

**Comment by chobitsfan** [comment]
Hi @Hwurzburg Thank you for helping

**Review by rmackay9 — APPROVED** [review]
One tiny typo but otherwise it's great, thanks so much!!

File: common/source/docs/common-vio-oak-d.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+- :ref:`EK3_SRC1_VELXY <EK3_SRC1_VELXY>` = 6 (ExternalNav)
+- :ref:`EK3_SRC1_POSZ <EK3_SRC1_POSZ>` = 1 (Baro which is safer)
+- :ref:`EK3_SRC1_VELZ <EK3_SRC1_VELZ>` = 0 (you can set it to 6 if 1st flight test looks good)
+- :ref:`EK3_SRC1_YAW <EK3_SRC1_YAW>` = 6 (ExternalNav)
+- :ref:`COMPASS_USE <COMPASS_USE>` = 0, :ref:`COMPASS_USE2 <COMPASS_USE2>` = 0, :ref:`COMPASS_USE3<COMPASS_USE3>` = 0 to disable all compasses
+
+How to run
+----------
+
+Open 3 terminals on PRI
```
[the remark this answers is not in the document]
Comment (chobitsfan) [reply common/source/docs/common-vio-oak-d.rst:65]: @rmackay9 Yes, thank you

**Comment by Hwurzburg** [comment]
fixed build error and squashed...merging...thanks

**Comment by chobitsfan** [comment]
Hi @Hwurzburg Thank you for fixing

**Pull request merged**